contrib/zpages/src/main/java/io/opencensus/contrib/zpages/ZPageHandlers.java

```diff
@@ -99,10 +94,14 @@
    * <p>If no sampled spans based on latency and error codes are available for a given name, make
    * sure that the span name is registered to the {@code SampledSpanStore}.
    *
+   * <p>When this method is called, {@link io.opencensus.trace.export.RunningSpanStore} will be
+   * enabled automatically.
+   *
    * @return a {@code ZPageHandler} for tracing debug.
    * @since 0.6
    */
   public static ZPageHandler getTracezZPageHandler() {
+    enableRunningSpanStore();
```

Comment:
Currently setting the max number of spans resets the store. Probably we should call enable only once.

Reply:
Make sense, updated in https://github.com/census-instrumentation/opencensus-java/pull/1878/commits/5d347352a5b7280acabaca1701609c30198b978d.